.gitignore

```diff
@@ -0,0 +1,6 @@
+test.md
+node_modules
+.stylelintrc.json
+.hintrc
+package-lock.json
+package.json
```

Comment:
- [ ] You should not ignore these files cause they are important to keep track of your project and add linters

Reply:
Thanks ! I have updated the .gitignore file